docs/external_cmps.md

```diff
@@ -0,0 +1,81 @@
+# External components
+
+By default, plain ```docker-compose up```, regardless of the
+pipeline will start containers for following services:
+
+* MySQL
+* Keystone
+* ElasticSearch
+* Zookeeper
+* Kafka
+
+However each of them is treated, in the same time, as the
```

Comment:
However**,** each of them is treated **** **at** the same time**** as the

Reply:
Done